Python/suggestions.c

```diff
@@ -1,3 +1,5 @@
+#define NEEDS_PY_IDENTIFIER
+
 #include "Python.h"
 #include "pycore_frame.h"
```

Comment:
```suggestion
#include "pycore_frame.h"
#include "pycore_runtime_init.h"  // _Py_ID()
```

Reply:
You would also have to regen the global strings.

Reply:
Will do this when I am back at home.